ini.go

```diff
@@ -48,6 +48,10 @@ var (
 	// at package init time.
 	LineBreak = "\n"
 
+	// Place custom spaces when close PrettyFormat and PrettyEqual
```

Comment:
s/when close PrettyFormat and PrettyEqual/when PrettyFormat and PrettyEqual are both disabled/